core/partitioning/partitioning.cpp

```diff
@@ -118,7 +118,7 @@ void setNonTensorConnectedNodes(PartitioningCtx* ctx, std::vector<torch::jit::No
       if (!isTensor(output)) {
         for (auto use : output->uses()) {
           auto node = use.user;
-          if (node->kind() != torch::jit::prim::Constant && ctx->shouldNodeRunInTensorRT(node)) {
+          if (isConstantOrUninitialized(node) && ctx->shouldNodeRunInTensorRT(node)) {
```

Comment:
Should this be `!isConstantOrUninitialized`?

Reply:
Yes, updated that just now. 
Thanks!